php/installer/Installer.class.inc

```diff
@@ -26,9 +26,16 @@ namespace LORIS\Installer;
  */
 class Installer
 {
-
     private $_lastErr;
 
+    private $_schemaFiles = ['00-schema.sql',
+        '01-Permission.sql',
+        '02-Modules.sql',
+        '03-ConfigTables.sql',
+        '04-Help.sql',
+        '05-ElectrophysiologyTables.sql'
+    ];
```

Comment:
This should probably be declared as a constant instead of a variable since it's not intended to change, but it's not a big deal